adcm_aio_client/core/errors.py

```diff
@@ -0,0 +1,47 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class RequesterError(Exception):
+    pass
+
+
+class SessionError(RequesterError):
+    pass
+
+
+class ResponseError(RequesterError):
+    pass
+
+
+class RequesterResponseError(ResponseError):
+    pass
```

Comment:
what's the difference?

Reply:
It is for errors in HTTPXRequesterResponse while processing data

Reply:
maybe distinct it from wrappers for 400+ http codes?

by hierarchy and by name?

for example inherit it from base client error (or requester base error) and name like `UnexpectedResponseDataType` or `ResponseDataConversionError`

Reply:
done